SAWHtoGit/GitExporter.cs

```diff
@@ -32,7 +32,7 @@ namespace SAWHtoGit
 	class GitExporter
 	{
 		FileStream ExportLog;
-		System.Text.Encoding Encoding = new System.Text.ASCIIEncoding();
+        System.Text.Encoding Encoding = new System.Text.ASCIIEncoding();

```

Comment:
```suggestion
		System.Text.Encoding Encoding = new System.Text.ASCIIEncoding();

```

Can you fix the indentation to be consistent with the rest of the file?